domain/account/repository-mock.go

```diff
@@ -15,8 +15,8 @@ type RepositoryMock struct {
 
 var _ Repository = RepositoryMock{}
 
-func (r RepositoryMock) CPFExists(cpf string) bool {
-	return r.CpfExists
+func (r RepositoryMock) CpfExists(cpf string) bool {
```

Comment:
```suggestion
func (r RepositoryMock) CPFExists(cpf string) bool {
```
é importante seguir as conveções da linguagem para [Initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)